keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -326,7 +326,18 @@ func checkKubernetesVersion(serverVersion *version.Info) error {
 //installKubeEdge downloads the provided version of KubeEdge.
 //Untar's in the specified location /etc/kubeedge/ and then copies
 //the binary to excecutables' path (eg: /usr/local/bin)
-func installKubeEdge(componentType types.ComponentType, arch string, version semver.Version) error {
+func installKubeEdge(componentType types.ComponentType, arch string, version semver.Version, packagePath string) error {
+	//create the storage path of the kubeedge installation packages
+	if packagePath == "" {
+		packagePath = KubeEdgePath
+	} else {
+		packagePath = packagePath + "/"
```

Comment:
Better to remove suffix, and add it when use.

Reply:
u are right:)